h2o-core/src/main/java/water/api/RapidsHandler.java

```diff
@@ -109,4 +111,21 @@ public InitIDV3 endSession(int version, InitIDV3 p) {
     }
     return p;
   }
+
+  public static class StartSession4 extends RestApiHandler<InputSchemaV4, SessionIdV4> {
```

Comment:
By our convention, `StartSessionV4` would be the name of the **schema** for a "start session" REST API call.
Whereas this class is not a schema, it's a handler. We could be calling it `StartSessionH4` for example, however there have been no convention as for naming of handlers from different rest versions.